### ISSUE TITLE
Feature reconstruction loss

### ISSUE THREAD

**Issue by zfjmike** [sniklaus/pytorch-sepconv#16]
Hi,

Did you post your code for feature reconstruction loss in this repo? It seems that the loss is very effective and important but I could not find the implementation.

Fangjun

**Comment by sniklaus** [comment]
I am afraid that we do not provide training code yet. Please note that this is not my decision and that I am having ongoing discussions with the involved stakeholders to change this. In the meantime, you could have a look at the feature loss in the [style transfer example](https://github.com/pytorch/examples/blob/master/fast_neural_style/neural_style/neural_style.py) in the PyTorch repository.

**Comment by zfjmike** [comment]
Got it! Thx a lot!
